doc/tutorials/gaussian_transformation.ipynb

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
<p> subset of CV operations: the <em>Gaussian  -&gt;</em> subset of CV operations: <em>Gaussian </em></p>

 _Reply via <a href='https://app.reviewnb.com/XanaduAI/pennylane/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>